recipe.yaml

```diff
@@ -40,5 +40,12 @@ Manifests:
                 -e SVCUID \
                 -v {kernel:rootPath}/ipc.socket:{kernel:rootPath}/ipc.socket \
                 greengrass_custom_secure_tunneling:NEXT_PATCH
+      Shutdown:
```

Comment:
`Shutdown:` should be paired with `Startup:`. Behavior of `Run:` and `Shutdown:` is undefined in the docs.